app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -1,18 +1,18 @@
-<p>Welcome, <%= @resource.name %>!</p>
-
 <% account_user = @resource&.account_users&.first %>
+<p>Welcome, <%= @resource.name %>!</p>
 <% if account_user&.inviter.present? && @resource.unconfirmed_email.blank? %>
-  <p><%= account_user.inviter.name %>, with <%= account_user.account.name %>, has invited you to try out <%= global_config['BRAND_NAME'] || 'Chatwoot' %>! </p>
+  <p><%= account_user.inviter.name %>, with <%= account_user.account.name %>, has invited you to try out <%= global_config['BRAND_NAME'] || 'Chatwoot' %>. Please click the link below to continue. </p>
 <% end %>
 
 <% if @resource.confirmed? %>
-<p>You can login to your account through the link below:</p>
+<p>Hi <%= @resource.name %>,</p>
```

Comment:
Why is there another welcome here? Don't we have it already at the top.

Reply:
Thanks for pointing out the mistake, I have moved it inside the if block. It was done as we were having multiple email copies within the same mail and the issue's copy changes are different from the existing one. This would make sure the exiting email copies does not change.